client/app/components/packages/pas-form/edit.hbs

```diff
@@ -63,7 +64,7 @@
 
         <Packages::PasForm::PasFormError @package={{@package}} />
         <pasForm.SubmitButton
-          @isEnabled={{pasForm.isSubmittable}}
+          @isEnabled={{and @package.isValid  pasForm.isSubmittable}}
```

Comment:
Doesn't `isSubmittable` imply `isValid`? A form isn't submittable if it isn't valid.

Reply:
There are two different types of validity -- Validator (ember-changeset) validity, and macro/entity-level validity for things like number of related entities

Reply:
Yeah but I think we're going to look at using ember changeset validations so that `isSubmittable` can capture validations on the # of models